Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -5083,6 +5092,14 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
+		1B3E93AA26E1649C00EA17B6 /* XCRemoteSwiftPackageReference "rust-components-swift" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/mozilla/rust-components-swift";
+			requirement = {
+				kind = upToNextMajorVersion;
+				minimumVersion = 0.0.3;
```

Comment:
Is 0.0.3 the latest? Since it was from a month ago, do we want to ship with a newer release? I was sort of expecting a version number here that matches the Glean and Nimbus versioning.

Reply:
yes it is, since swift packages are fairly new -- `rust-components-swift` actually will contains ALL the components of application services (eventually). Which does bring up a good point that eventually I might need to change this to the versioning of `rust-components-swift` to match application-services.